data/sql_incremental_aggregates/prepare_schedule_a_aggregate_employer.sql

```diff
@@ -14,60 +14,17 @@ and is_individual(contb_receipt_amt, receipt_tp, line_num, memo_cd, memo_text, c
 group by cmte_id, cycle, employer
 ;
 
-alter table ofec_sched_a_aggregate_employer_tmp add column idx serial primary key;
+alter table ofec_sched_a_aggregate_employer_tmp add column idx serial;
+alter table ofec_sched_a_aggregate_employer_tmp add constraint uq_cmte_id_cycle_employer unique (cmte_id, cycle, employer);
 
 -- Create indices on aggregate
-create index on ofec_sched_a_aggregate_employer_tmp (cmte_id, idx);
-create index on ofec_sched_a_aggregate_employer_tmp (cycle, idx);
-create index on ofec_sched_a_aggregate_employer_tmp (employer, idx);
-create index on ofec_sched_a_aggregate_employer_tmp (total, idx);
-create index on ofec_sched_a_aggregate_employer_tmp (count, idx);
+create index on ofec_sched_a_aggregate_employer_tmp (cmte_id);
+create index on ofec_sched_a_aggregate_employer_tmp (cycle);
+create index on ofec_sched_a_aggregate_employer_tmp (employer);
+create index on ofec_sched_a_aggregate_employer_tmp (total);
+create index on ofec_sched_a_aggregate_employer_tmp (count);
```

Comment:
Also noting that we may have to readjust these again based on the findings and the fact that the `idx` column is currently leveraged in the application code - given that we had to create those compound indexes for the itemized schedule data, would we have to maintain that here, or create different compound indexes?  Or would we be able to leverage PostgreSQL itself knowing what to combine, which [the documentation](https://www.postgresql.org/docs/9.6/static/indexes-multicolumn.html) says is usually more than sufficient?